{{cookiecutter.project_slug}}/.neuro.toml

```diff
@@ -1,2 +1,3 @@
 [storage]
-cp-exclude = ["*.pyc", ".mypy_cache", "__pycache__", ".ipynb_checkpoints"]
+cp-exclude = [".git", "*.pyc", ".mypy_cache", "__pycache__", ".ipynb_checkpoints", ".neuro"]
```

Comment:
minor: let's just keep `[".git", ".neuro"]` because everything else is in gitignore.

Reply:
Goot catch, changed in 3524734